CHANGELOG.rst

```diff
@@ -2,8 +2,9 @@
 CHANGELOG
 =========
 
-Unreleased
+2.10.0
 ==========
+* bugfix: Defensively copy context entities to async thread. `PR340 <https://github.com/aws/aws-xray-sdk-python/pull/340>`_.
 * improvement: Added support for IGNORE_ERROR option when context is missing. `PR338 <https://github.com/aws/aws-xray-sdk-python/pull/338>`_.
```

Comment:
We should also include this PR: https://github.com/aws/aws-xray-sdk-python/pull/343